Add a case when CType migration has not been done (#188)
TYPO3-Documentation/TYPO3CMS-Exceptions: Documentation/Exceptions/1540246570.rst

..  code-block:: xml

    <f:translate key = "LLL:EXT:yourExtension/Resources/Private/Language/locallang.xml:Button_Helpful" />

> TYPO3 12/13 - General Plugin: No Content Object definition found at TypoScript object path "tt_content.list.20

Reviewer: ```suggestion
TYPO3 12/13 - General Plugin: No Content Object definition found at TypoScript object path "tt_content.list.20"
```

==============================================================================================================

> This exception could happen because a plugin were registered before using the :php:`list` content element and the plugin subtype :php:`list_type` and are now registered as :php:`ExtensionUtility::PLUGIN_TYPE_CONTENT_ELEMENT`.

Reviewer: ```suggestion
This exception could happen because a plugin was registered before using the :php:`list` content element and the plugin subtype :php:`list_type` and is now registered as :php:`ExtensionUtility::PLUGIN_TYPE_CONTENT_ELEMENT`.
```
